renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -1245,6 +1254,7 @@ void EmulateRequiredExtensions(const GLHookSet *real, GLHookSet *hooks)
     EMULATE_FUNC(glGetNamedFramebufferAttachmentParameterivEXT);
     EMULATE_FUNC(glGetNamedBufferParameterivEXT);
     EMULATE_FUNC(glCheckNamedFramebufferStatusEXT);
+    EMULATE_FUNC(glGetBufferSubData);
```

Comment:
This shouldn't be in `if(HasExt[EXT_direct_state_access])`. Given that it's not available at all on gles but is always available on desktop GL, it should be in a separate `if(IsGLES)` I think - that's a global added by #532 which I'm merging now.